docs/en/ingest-management/integrations/upgrade-integration.asciidoc

```diff
@@ -0,0 +1,147 @@
+connect to `https://epr.elastic.co` on port `443`.
+
+//QUESTION: Do we want to mention air gapped environments here?
+
+Elastic releases {agent} integration updates periodically. To use new features
+and capabilities, upgrade the installed integration to the latest version and
+optionally upgrade integration policies to use the new version.
+
+TIP: In larger deployments, you should test integration upgrades on a sample
+{agent} before rolling out a larger upgrade initiative.
+
+[discrete]
+[[upgrade-integration-to-latest-version]]
+== Upgrade an integration to the latest version
+
+//TODO: Make sure Integrations is added back into the Management section in 7.16.
+//Otherwise, change this step to match the UI.
+
+. In {kib}, go to *Management > Integrations > Installed*. Search for and select
+the integration you want to upgrade. Notice there is a warning icon next to the
+version number to indicate a new version is available.
+
+. Click the *Settings* tab and notice the message about the new version.
++
+[role="screenshot"]
+image::images/upgrade-integration.png[Settings tab under Integrations shows how to upgrade the integration]
+
+. Before upgrading the integration, decide whether to upgrade integration
+policies to the latest version, too. To use new features and capabilities,
+you'll need to upgrade existing integration policies. However, the upgrade may
+introduce changes, such as field changes, that require you to resolve conflicts.
++
+--
+* Select *Upgrade integration policies* for {fleet} to upgrade any eligible
+integration policies when it upgrades the integration.
+
+* To continue using the older package version, deselect
+*Upgrade integration policies*. You can still choose to
+<<upgrade-integration-policies-manually,upgrade integration policies manually>>
+later on.
+--
+
+. Click *Upgrade to latest version*.
++
+If you selected *Upgrade integration policies* and there are conflicts,
+<<upgrade-integration-policies-manually,upgrade integration policies manually>>
+and resolve the conflicts in the policy editor.
+
+. After the upgrade is complete, verify that the installed version and latest
+version match.
+
+[discrete]
+[[upgrade-integration-policies-automatically]]
+== Keep integration policies up to date automatically
+
+Some internal integration packages, like System, are upgraded automatically when
+{fleet} detects that a new version is available. These integrations have an
+option to upgrade integration policies automatically, too. When this option is
+selected (the default), {fleet} automatically upgrades your policies behind the
+scenes when a new version of the integration is available. If there are
+conflicts during the upgrade, your integration policies will not be upgraded,
+and you'll need to
+<<upgrade-integration-policies-manually,upgrade integration policies manually>>.
+
+//QUESTION: Is it true that this setting on by default for all internal integrations? 
```

Comment:
This is true for all of Fleet's "default" packages that are installed during setup. So that list includes the `system`, `elastic_agent` and `fleet_server` integration packages.

<img width="1278" alt="Screen Shot 2021-11-16 at 8 05 54 AM" src="https://user-images.githubusercontent.com/6766512/141990575-f086a1c8-567a-4710-8726-da825ea6a475.png">

I'm not sure if we reference this specific set of packages by any name elsewhere in the docs, but we could refer to them here as "Fleet's default integration packages" or "integration packages that are installed by default" something along those lines?